library/src/main/java/org/xmtp/android/library/Client.kt

```diff
@@ -670,11 +726,11 @@ class Client() {
     }
 
     val privateKeyBundle: PrivateKeyBundle
-        get() = PrivateKeyBundleBuilder.buildFromV1Key(privateKeyBundleV1)
+        get() = PrivateKeyBundleBuilder.buildFromV1Key(v1keys)
 
     val v1keys: PrivateKeyBundleV1
-        get() = privateKeyBundleV1
+        get() = privateKeyBundleV1 ?: throw XMTPException("V2 only function")
 
     val keys: PrivateKeyBundleV2
-        get() = privateKeyBundleV1.toV2()
+        get() = v1keys.toV2()
 }
```

Comment:
logic all seems fine here, but these names seem very confusing for me personally. The names aren't new in this PR though, so no need to block the change. 

an example improvement might be:
( we already have the nullable class field `privateKeyBundleV1`)
- `v1keys` could instead be called something like `nonNullPrivateKeyBundleV1`
- `keys` could instead be called `privateKeyBundleV2`

Reply:
Ya this one really thew me off as well something I will be happy to kill with V2.